app/views/home/about.html.erb

```diff
@@ -0,0 +1,74 @@
+<div class="container about-page">
+  <div class="row about-intro">
+
+      <div class="col-md-12">
+          <h1 class="intro-title">We made Giving Profiles to help you help others</h1>
+          <p class="about-paragraph">Giving Profiles is a simple way to collect all your charitable donations in a single place. Use it to keep track of your giving, share your favorite charities with friends, and discover great new organizations to support. We are a team of researchers from Harvard and Yale who specialize in motivating people to do more good. We're not venture capitalists, or serial-startup-ists, and we're not here to make a buck. We're here to put our research into practice, and see if we can help you make the biggest difference you can.</p>
+          <hr class="line-one">
+      </div>
+
+   </div>
+
+   <div class="row team-members">
+
+      <div class="col-md-2">
+        <h2 class="about-row-two-header">OUR TEAM</h2>
```

Comment:
Might be worth taking a BEM approach to everything here too. I think it could be good to do a quick clinic on Thursday after the call.